examples/cronjob_crud.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python3
+# -*- coding:utf-8 -*-
+
+import json
+import time
+
+from kubernetes import client, config
+
+config.load_kube_config()
+
+
+def create_namespaced_cron_job(namespace='default', body=None):
+    cronjob_json = body
+    if body is None:
+        print('body is required!')
+        exit(0)
+    name = body['metadata']['name']
+    if judge_crontab_exists(namespace, name):
+        print(f'{name} exists, please do not repeat!')
+    else:
+        v1 = client.BatchV1beta1Api()
```

Comment:
https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/

Please use batch/v1

Reply:
thanks, I have used batch/v1 in the new commit